docs/software-development-kits/dotnet/dotnetcore.mdx

```diff
@@ -324,12 +438,11 @@ To enable Topaz authorization, you need to add a dependency to the Aserto dotnet
 $dotnetmvc/src/QuickstartMVC$ dotnet add package Aserto.AspNetCore.Middleware
 ```
 
-Configure the Authorizer API Key, Tenant ID, Policy root and Policy ID in the `appsettings.json`:
+Configure the Policy root and Policy ID in the `appsettings.json`:
```

Comment:
Is `Policy ID` correct, we no longer use policy IDs, nor do I see it in the appsettings?